exasol_integration_test_docker_environment/test/test_hash_temp_dir.py

```diff
@@ -78,25 +88,25 @@ def test_file_with_path(self):
                                          use_relative_paths=False,
                                          hash_directory_names=True,
                                          hash_file_names=True)
-        hash = hasher.hash([TEST_FILE])
+        hash = hasher.hash([simple_path_mapping(TEST_FILE)])
         ascii_hash = base64.b32encode(hash).decode("ASCII")
-        self.assertEqual("AOBF7HELTYAPHBEW6HQQ74N3BKGSNZTJXB4MTOEROHO5VH6YYJOA====", ascii_hash)
+        self.assertEqual("7D34CBUU2SNSWF3UFM6A7BYFJVV5ZFEY5F6THIMGJY725WC45KEA====", ascii_hash)
 
     def test_directory_with_relative_paths_fixed_hash(self):
         hasher = FileDirectoryListHasher(hashfunc="sha256",
                                          use_relative_paths=True,
                                          hash_directory_names=True,
                                          hash_file_names=True)
-        hash = hasher.hash([self.test_dir1])
+        hash = hasher.hash([simple_path_mapping(self.test_dir1)])
         ascii_hash = base64.b32encode(hash).decode("ASCII")
-        self.assertEqual("VIN3VCPDX7DAC4GD37IDF4KQTCDNNH72QV5PARVGGQ4OMB4DZTLA====", ascii_hash)
+        self.assertEqual("EZ3ER6KZHAAYG4JNFGFLHUI7TVHZVIRVOV4QWJT4ERQ4XGI2GLUA====", ascii_hash)
 
     def test_directory_content_only_fixed_hash(self):
         hasher = FileDirectoryListHasher(hashfunc="sha256",
                                          use_relative_paths=False,
                                          hash_directory_names=False,
                                          hash_file_names=False)
-        hash = hasher.hash([self.test_dir1])
+        hash = hasher.hash([PathMapping("level0", f"{self.test_dir1}/level0")])
```

Comment:
O think, this should be actually the same, with the original parameter, because it only hashes the file content

Reply:
That's true. Reverted it.